src/Layers/xrRender/dxFontRender.cpp

```diff
@@ -101,7 +101,7 @@ void dxFontRender::OnRender(CGameFont& owner)
                     clr2 = color_rgba(_R, _G, _B, _A);
                 }
 
-#ifndef USE_DX9 // Vertex shader will cancel a DX9 correction, so make fake offset
+#if defined(USE_DX11) || defined(USE_OGL) // Vertex shader will cancel a DX9 correction, so make fake offset
```

Comment:
Special case for DX9 only.
```suggestion
#ifndef USE_DX9 // Vertex shader will cancel a DX9 correction, so make fake offset
```